docs/devguide/newdashboards.asciidoc

```diff
@@ -297,6 +210,61 @@ If Elasticsearch is running on a different host, then you can use the `ES_URL` v
 ES_URL="http://192.168.3.206:9200" make export-dashboards
 ----------------------------
 
+/////////////////////
+
+==== Exporting Kibana 6.0 dashboards and newer
+
+The `dev-tools/cmd/export_dashboards.go` script helps you export your customized Kibana 6.0 dashboards and newer. You might need to export a single dashboard 
```

Comment:
Is there a thought missing at the end of this sentence? There is no period, and it just seems to drop off.

Reply:
done, thanks.